src/Satooshi/Bundle/CoverallsBundle/Api/Jobs.php

```diff
@@ -96,7 +96,7 @@ public function collectEnvVars(array $env)
 
         try {
             $this->jsonFile->fillJobs($envCollector->collect($env));
-        } catch (\Satooshi\Bundle\CoverallsV1Bundle\Entity\Exception\RequirementsNotSatisfiedException $e) {
+        } catch (\Satooshi\Bundle\CoverallsBundle\Entity\Exception\RequirementsNotSatisfiedException $e) {
```

Comment:
@keradus 

Unrelated to this PR and the entire issue, but what do you think about dropping FQCNs here and in other places as a follow-up to #225 before tagging?